app/src/main/java/org/traccar/client/MainActivity.java

```diff
@@ -307,4 +291,47 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
+    private boolean setServerURL(String userUrl) {
+        EditTextPreference preference = (EditTextPreference) findPreference(KEY_URL);
+
+        if (userUrl == null || userUrl.trim().length() == 0) {
+            preference.setText(getString(R.string.settings_url_default_value));
+            preference.setSummary(R.string.settings_url_summary);
+            findPreference(KEY_STATUS).setEnabled(true);
```

Comment:
That's not how validation worked before, and it still works differently for other parameters. I think we should be consistent and use original way of validating parameters.